nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -351,89 +129,13 @@ def forward(self, x, *args):
     slice_list.append(slice_obj)
     return SliceMoudle(tuple(slice_list))
 
+def tupleunpack_python(node, speedup):
```

Comment:
Unused parameter should be prefixed with underscore.

Reply:
ok